backend/src/main/java/edu/bachelor/rest/controller/StoredFileController.java

```diff
@@ -69,14 +67,20 @@ public ResponseEntity<byte[]> previewStoredFileById(@PathVariable Long id) throw
   }
 
   @GetMapping("/owner/{owner_id}")
-  public List<StoredFileDTO> getStoredFileByOwnerId(@PathVariable Long owner_id) {
-    return this.storedFileService.getStoredFilesByOwnerId(owner_id);
+  public List<StoredFileDTO> getStoredFileByOwnerId(
+      @PathVariable Long owner_id,
+      @RequestParam(name = "fetchContent", defaultValue = "0") boolean fetch_content) {
```

Comment:
defaultValue = 0 while being boolean? looks like an issue to me


```suggestion
      @RequestParam(name = "fetchContent", defaultValue = "false") boolean fetch_content) {
```

Reply:
In fact, there is a problem with it, it worked so I missed it.